source/plugin/workspace/idea.rst

```diff
@@ -2,63 +2,83 @@
 Setting Up IntelliJ IDEA
 ========================
 
-This article describes how to configure your **IntelliJ IDEA** workspace for plugin development with the Sponge API.
-
-Starting Your Project
-=====================
-
-We recommend using Gradle or Maven when developing plugins with the Sponge API. Gradle and Maven help significantly when
-managing dependencies, such as the Sponge API, for your project. This is preferable to including the ``.jar`` file in
-your project.
+This article describes how to configure your **IntelliJ IDEA** workspace for plugin development with SpongeAPI and
+:doc:`a build system such as Maven or Gradle <../buildsystem/>`.
 
 Gradle
-~~~~~~
+======
 
-**To create your project:**
+Creating your project
+~~~~~~~~~~~~~~~~~~~~~
 
 * Open **IntelliJ IDEA**.
 * Click ``Create New Project``.
 * Select ``Gradle`` in the popup, and click ``Next``.
 * Enter your **Group ID**, **Artifact ID**, and **Version**.
 
-  * Your **Group ID** should correspond to your organization name, or something similar.
-  * Your **Artifact ID** should correspond to your project name.
+  * Your **Group ID** should usually correspond to your the **plugin group ID** you chose earlier, e.g.
+    ``io.github.username``.
+  * Your **Artifact ID** should usually correspond to your **unqualified plugin ID** you chose earlier, e.g.
+    ``myplugin``.
   * Your **Version** is up to you.
 
 * Click ``Next`` twice, name your project, and click ``Finish``.
 
-**To import the Sponge API as a dependency:**
+Editing the build script
+~~~~~~~~~~~~~~~~~~~~~~~~
 
 * Open ``build.gradle`` in the navigator and add the dependencies.
-* Have a look at the :doc:`dependencies page <dependencies>` for an example Gradle file.
+* Edit the build script according to the instructions at TODO.
 * Open the **Gradle tab** on the right of the IntelliJ window and hit the refresh button.
 * Gradle setup is done! Now you can start coding your plugin.
 
+Importing your project
+~~~~~~~~~~~~~~~~~~~~~~
+
+If you've already started with your project and want to import it again at a later point you need to import it instead
+of re-creating it inside your IDE:
+
+.. TODO
```

Comment:
To import a project in IntelliJ, simply click File -> Open, and navigate to the root folder of your project. Click 'Ok', and it'll start the import process.

Alternatively, if a build system was used, it is recommended to navigate to the build.gradle, or pom.xml for gradle and maven respectively.